charts/alfresco-connector-ms365/values.yaml

```
@@ -43,3 +43,5 @@ repository:
   host: acs-alfresco-cs-repository
   # -- ACS repository port
   port: 80
+  # -- a pre-existing configmap which provides expected configuration for ms365
+  existingConfigMap:
```

```suggestion
  existingConfigMap: null
```

or updatecli will complain later :D